src/execution.rs

```diff
@@ -49,6 +49,7 @@ pub fn execute_abi_scs(
     pb.finish_print("Finished executing ABI SCs");
 }
 
+#[allow(dead_code)]
```

Comment:
why this?

Reply:
It's to execute the WAT smart contracts, which are commented out in main :)

Reply:
A note relative to this: if we need to gas-calibrate the WAT smart contracts, a small fix can be to add a 0x02 in front of them, and change the runtime's behaviour to strip the 0x02 byte and pass it to the ASCompiler.